Src/GoogleApis.Core/Apis/Requests/Parameters/ParameterUtils.cs

```diff
@@ -29,6 +30,9 @@ namespace Google.Apis.Requests.Parameters
     /// </summary>
     public static class ParameterUtils
     {
+        //TODO: Find out how to resolve the issue with setting up the Logger:
+        //private static readonly ILogger Logger = ApplicationContext.Logger.ForType<ParameterUtils>();

```

Comment:
I think it should work if you add a static constructor and set the Logger there.
What is the exact error you get?